masa/validator/forwarder.py

```diff
@@ -94,3 +170,36 @@ def sanitize_responses_and_uids(self, responses, miner_uids):
             if response is not None
         ]
         return valid_responses, valid_miner_uids
+
+    async def get_source_of_truth(self, responses, miner_uids, source_method, query):
+        responses_str = [str(response) for response in responses]
+        weighted_responses = defaultdict(float)
+        most_common_response = None
+        count_high_score_uids = sum(
+            1 for uid in miner_uids if self.validator.scores[uid] > self.minimum_accepted_score)
```

Comment:
should make this `>=` instead of `>` if variable is named "miinimum_accepted_score"

Reply:
```suggestion
            1 for uid in miner_uids if self.validator.scores[uid] >= self.minimum_accepted_score)
```